romancal/tweakreg/tests/test_tweakreg.py

```diff
@@ -627,96 +628,6 @@ def test_tweakreg_updates_group_id(tmp_path, base_image):
         res.shelve(model, 0, modify=False)
 
 
-@pytest.mark.parametrize(
-    "shift_1, shift_2, tolerance, is_small_correction",
-    [
-        (0, 0, 5, True),
-        (0, 3, 5, True),
-        (1, 1, 5, True),
-        (5, 5, (5**2 + 5**2) ** 0.5, True),
-        (5, 5, 5, False),
-        (5, 5, 1, False),
-        (5, 5, 3, False),
-        (5, 10, 5, False),
-    ],
-)
-def test_tweakreg_correction_magnitude(
```

Comment:
Is this no longer needed? I think it's checking if small wcs corrections are correctly identified. This looks to be handled by stcal: https://github.com/spacetelescope/stcal/blob/c3e133228528f369eb85d34c81e2ceb4c76eaaff/src/stcal/tweakreg/tweakreg.py#L101C20-L101C44
If this is correct, no objection from me for removing this.

Reply:
Yes, since we moved it to `stcal`, there's no need to keep it around anymore.
The current `stcal` test for that is here: https://github.com/spacetelescope/stcal/blob/c3e133228528f369eb85d34c81e2ceb4c76eaaff/tests/test_tweakreg.py#L136-L148